mip/model.py

```diff
@@ -4,6 +4,7 @@
 from typing import List, Tuple, Optional, Union, Dict, Any
 import numbers
 import mip
+import numpy as np
```

Comment:
already imported in the try except below, to make numpy optional

Reply:
Yes, you can remove the numpy import in the beginning